addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -407,23 +418,15 @@ function selectElementInWeSelectWidget(
             run: `text ${elementName}`,
         });
     }
+    steps.push(clickOnElement(`${elementName} in the ${widgetName} widget`,
+        `we-select[data-name=${widgetName}] we-button:contains(${elementName})`));
     steps.push({
-        content: `Clicking on the ${elementName} in the ${widgetName} widget`,
-        trigger: `${we_select} we-button:contains("${elementName}")`,
-        run: "click",
-    });
-    steps.push({
-        content:
-            "Check we-select is set and wait a delay before continue the tour",
-        trigger: `${we_select}:contains(${elementName})`,
+        content: "Check we-select is set",
+        trigger: `we-select[data-name=${widgetName}]:contains(${elementName})`,
         async run() {
-            // fix underterministic error.
-            // When we-select is used twice a row too fast, the second we-select may not open.
-            // The first toggle is open, we click on it and almost at the same time, we click on the second one.
-            // There may be confusion with the active class.
-            // Add a delay before continue the tour solves this problem.
+            // TODO: remove this delay. Will be fixed in 18.0
```

Comment:
explain why there's a delay?

Reply:
Yes, I'd say the previous comment was good, it just needed an extra TODO, thanks :+1: